docs/README.md

```diff
@@ -70,17 +97,76 @@ To access the dashboard, a signed-in Campaign Organizer should click on his user
 
 Within the Dashboard, a user can view all the Campaigns, Collections, Vocabularies and Usergroups - each under the respective tab - as well as create new ones or edit existing ones.
 
+<img src="_media/dashboard-editors.png" width="auto">
+
 ### Create/Edit Campaign
 To create a new Campaign, click on the *+ New Campaign* button on the top right of the **Campaigns** tab (as seen on the above screenshot). A sidebar will appear on the right, asking to choose the Campaign's shortname. If the shortname already exist, you will be notified to select another.
 
 <img src="https://raw.githubusercontent.com/ails-lab/CrowdHeritage_Documentation/main/docs/_media/image12.jpg" width="auto">
 
-Once you click on the *Create* button, your new Campaign will appear on the Campaigns list and you can click on it to edit any or all of the details. There is a long list of input fields like: Visibility, Start/End Dates, Input Language, Banner, Description and so on. There are buttons to *Preview* and/or *Save* the Campaign after you are done.
+Once you click on the *Create* button, your new Campaign will appear on the Campaigns list and you can click on it to edit any or all of the *Campaign Details*. There you also have multiple configuration options via the *Campaign Annotating*, *Campaign Access* and *Campaign Base Annotations* sections to customise the campaign according to your needs.
+
+#### Campaign Details Section
+
+This is a long list of input fields like: Visibility, Start/End Dates, Input Language, Banner, Description and so on. There are buttons to *Preview* and/or *Save* the Campaign after you are done.
 
 <img src="https://raw.githubusercontent.com/ails-lab/CrowdHeritage_Documentation/main/docs/_media/image13.jpg" width="auto">
 
 The input fields marked as *(multilingual)* can be edited in multiple languages by selecting their appropriate language on the **Input Language** dropdown menu on top. This way, whenever a user views a Campaign, it will appear in their own language if the Campaign Owner has added the relevant text in the user's language.
 
+#### Campaign Annotating Section
+
+First, the user needs to select one of the available campaign types (basic, translate, image comparison) through the *campaign type* option.
+Based on the campaign type different options become availablem.
```

Comment:
Proposed changes:
Based on the **selected** campaign type different **campaign parameters** become **available** (typo).